.travis.yml

```diff
@@ -13,6 +13,10 @@ android:
   licenses:
     - ".+"
 
+before_install:
+  - mkdir -p "$ANDROID_HOME/licenses/"
+  - curl "https://download.msfjarvis.website/.viscerion/sdk-licenses.tar.xz" | tar -C "$ANDROID_HOME/licenses/" --strip-components=1 -xvf -
```

Comment:
Is there a way to do this without downloading from your personal website?

Reply:
Yes, I'll be changing that, just copied a quick-fix from an internal repo to resolve CI failure.

Reply:
Done with 0971a72